2
[1] "2"
Loading required package: iterators
starting worker pid=23496 on localhost:11827 at 14:24:47.952
starting worker pid=23499 on localhost:11827 at 14:24:47.954
starting worker pid=23495 on localhost:11827 at 14:24:47.971
starting worker pid=23497 on localhost:11827 at 14:24:47.978
starting worker pid=23491 on localhost:11827 at 14:24:47.985
starting worker pid=23488 on localhost:11827 at 14:24:47.997
starting worker pid=23489 on localhost:11827 at 14:24:47.998
starting worker pid=23494 on localhost:11827 at 14:24:47.998
starting worker pid=23492 on localhost:11827 at 14:24:48.000
starting worker pid=23503 on localhost:11827 at 14:24:48.001
starting worker pid=23500 on localhost:11827 at 14:24:48.000
starting worker pid=23501 on localhost:11827 at 14:24:48.002
starting worker pid=23498 on localhost:11827 at 14:24:48.003
starting worker pid=23490 on localhost:11827 at 14:24:48.006
starting worker pid=23502 on localhost:11827 at 14:24:48.009
starting worker pid=23493 on localhost:11827 at 14:24:48.009
discovered package(s): 
automatically exporting the following variables from the local environment:
  iter, l_tx 
explicitly exporting package(s): PanicModel
numValues: 16, numResults: 0, stopped: TRUE
got results for task 1
numValues: 16, numResults: 1, stopped: TRUE
returning status FALSE
got results for task 2
numValues: 16, numResults: 2, stopped: TRUE
returning status FALSE
got results for task 3
numValues: 16, numResults: 3, stopped: TRUE
returning status FALSE
got results for task 4
numValues: 16, numResults: 4, stopped: TRUE
returning status FALSE
got results for task 5
numValues: 16, numResults: 5, stopped: TRUE
returning status FALSE
got results for task 6
numValues: 16, numResults: 6, stopped: TRUE
returning status FALSE
got results for task 7
numValues: 16, numResults: 7, stopped: TRUE
returning status FALSE
got results for task 8
numValues: 16, numResults: 8, stopped: TRUE
returning status FALSE
got results for task 9
numValues: 16, numResults: 9, stopped: TRUE
returning status FALSE
got results for task 10
numValues: 16, numResults: 10, stopped: TRUE
returning status FALSE
got results for task 11
numValues: 16, numResults: 11, stopped: TRUE
returning status FALSE
got results for task 12
numValues: 16, numResults: 12, stopped: TRUE
returning status FALSE
got results for task 13
numValues: 16, numResults: 13, stopped: TRUE
returning status FALSE
got results for task 14
numValues: 16, numResults: 14, stopped: TRUE
returning status FALSE
got results for task 15
numValues: 16, numResults: 15, stopped: TRUE
returning status FALSE
got results for task 16
numValues: 16, numResults: 16, stopped: TRUE
calling combine function
evaluating call object to combine results:
  fun(accum, result.1, result.2, result.3, result.4, result.5, 
    result.6, result.7, result.8, result.9, result.10, result.11, 
    result.12, result.13, result.14, result.15, result.16)
returning status TRUE
Warning message:
In e$fun(obj, substitute(ex), parent.frame(), e$data) :
  already exporting variable(s): l_tx
[1] "Full Timing Iteration 2:"
elapsed 
19596.8 
